pages/models/regulations/AC1C/index.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: Standard voltage regulator model
+tags: ["Voltage regulator", "AC1C", "generic"]
+---
+# AC1C model
+
+## Context
+
+This voltage regulator model has been developed by RTE.
+
+## Model inputs and output
+
+The input variables are the measured and reference stator voltages, the rotor current and possibly the output voltages of the power system stabilizer, the underexcitation limiter, the overexcitation limiter and the stator current underexcitation and overexcitation limiters.
```

Comment:
can you put the name of input variables?

Reply:
Done.